wheels/model/adapters/MySQL.cfc

```diff
@@ -50,9 +50,12 @@ component extends="Base" output=false {
 			case "varbinary":
 				local.rv = "cf_sql_varbinary";
 				break;
-			case "varchar": case "text": case "mediumtext": case "longtext": case "tinytext": case "enum": case "set":
+			case "varchar": case "enum": case "set":
 				local.rv = "cf_sql_varchar";
 				break;
+			case "json": case "text": case "mediumtext": case "longtext": case "tinytext": 
+                loc.rv = "cf_sql_longvarchar";
```

Comment:
For CFWheels 2.0, this should be using the `local` scope, not the now-defunct `var`ed `loc` struct.